### PR TITLE
Draw Effects within the Layer Properties -> Style

### PR DESCRIPTION
**Pull request by purplexed**
Wrote about the options of the draw effects, and also included screenshots for each one of them.

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -667,13 +667,109 @@ features of the layer:
...
+    * **Inner Shadow:** This effect is similar to the :menuselection:'Drop Shadow' effect,
+    but it adds the shadow effect on the inside of the edges of the feature. The
+    available options for customization are the same as the :menuselection:'Drop Shadow'
+    effect.
+    .. figure:: /static/user_manual/working_with_vector/inner_shadow.png
+       :align: center
+
+       Draw Effects: Inner Shadow dialog |nix|
+
+    * **Outer Glow:** This effect is similar to the :menuselection:'Inner Glow' effect,
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: :menuselection:'Inner Glow'should be :menuselection:`Inner Glow`

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -667,13 +667,109 @@ features of the layer:
...
+
+    * **Inner Shadow:** This effect is similar to the :menuselection:'Drop Shadow' effect,
+    but it adds the shadow effect on the inside of the edges of the feature. The
+    available options for customization are the same as the :menuselection:'Drop Shadow'
+    effect.
+    .. figure:: /static/user_manual/working_with_vector/inner_shadow.png
+       :align: center
+
+       Draw Effects: Inner Shadow dialog |nix|
+
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: There should be an empty line before .. figure... line.
And more globally, the insertion of images should follow the guideline (http://docs.qgis.org/testing/en/docs/documentation_guidelines/README.html#figure-and-image)
And you can get rid of the use of |nix|, now.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -667,13 +667,109 @@ features of the layer:
...
+    turns the feature around its center point and the :menuselection:'Translate' effect
+    changes the position of the item based on a distance given on the x and/or the y axis.
+    .. figure:: /static/user_manual/working_with_vector/transform.png
+       :align: center
+
+       Draw Effects: Transform dialog |nix|
+
+     In each of the Draw effect types, the :menuselection:'Blend mode' and :menuselection:'Draw mode'
+     can be adjusted to match the user needs. The Blend mode follows the same methods as the ones
+     included for the layers (link here) and cannot be used for the transform effect.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: you can use :ref:`blend-modes` to link to http://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/vector_properties.html#blending-modes. More information at https://github.com/qgis/QGIS-Documentation/tree/master/source/docs/documentation_guidelines#labelreference

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 4]
```diff
@@ -667,13 +667,109 @@ features of the layer:
...
+    can change are the :menuselection:'Blur type' (:menuselection:'Stack' or
+    :menuselection:'Gaussian blur'), the strength and transparency of the blur
+    effect.
+    .. figure:: /static/user_manual/working_with_vector/blur.png
+       :align: center
+
+       Draw Effects: Blur dialog |nix|
+
+    * **Colorize:** This effect can be used by someone who wants to adjust the
+    :menuselection:'brightness', :menuselection:'contrast' and :menuselection:'saturation'
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: I may be wrong but these should be :guilabel: tags followed with the same ` character

**Comment by DelazJ** [comment]
detailed and nice description.
Thanks

**Comment by purplexed** [comment]
@yjacolin @DelazJ Thank you both for your input. Will do the changes and place a PR again.

**Comment by yjacolin** [comment]
hum why did you change all ' caracters by `?

**Comment by yjacolin** [comment]
I added some comment into your commit to show you where it should be changer. In a quick way: you should use simple quout (') in a normal sentence (`it's fine` for example). There are some special tag as :menuselection: (and others) which need a value. This value should be between back quote : ```:menuselection:`my selection````

Hopes it will help you.

**Comment by purplexed** [comment]
@yjacolin Thank you very much for looking into these. I reversed the file back to the original, and rewrote the Draw Effects part (just in case I missed anything). I hope it will come good now. Apologies for the many errors, it is my first time I contribute in such a big project.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 5]
```diff
@@ -667,11 +667,132 @@ features of the layer:
...
 * :guilabel:`Layer blending mode` and :guilabel:`Feature blending mode`: You can
   achieve special rendering effects with these tools that you may
   previously only know from graphics programs. The pixels of your overlaying and
   underlaying layers are mixed through the settings described in :ref:`blend-modes`.
-* :guilabel:`Draw effects`
+
+* :guilabel:`Draw effects`: This function adds paint effects for customizing the
+  visualization of vector files. The function is activated by checking the
+  :menuselection:`Draw effects` button and then customize it by clicking the
+  :menuselection:`Customize effects` button (the small “star” shape, on the right
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: is it windows quotes?  `“star”`, could you use "star" instead?

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 6]
```diff
@@ -667,11 +667,132 @@ features of the layer:
...
+  of the :menuselection:`Draw effects` button).
+  The effects include the following categories, with the following options:
+    * **Blur:** Adds a blur effect on the vector layer. The options that someone
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: I think we need a blanck line before.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 7]
```diff
@@ -667,11 +667,132 @@ features of the layer:
...
+       :align: center
+
+       Draw Effects: Inner Glow dialog
+
+    * **Inner Shadow:** This effect is similar to the :menuselection:`Drop Shadow` effect,
+    but it adds the shadow effect on the inside of the edges of the feature. The
+    available options for customization are the same as the :menuselection:`Drop Shadow`
+    effect.
+
+    **Figure Symbology 13:**
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Test failed here but I can't find why.
```
Warning, treated as error:

/home/travis/build/qgis/QGIS-Documentation/source/docs/user_manual/working_with_vector/vector_properties.rst:751: ERROR: Unexpected indentation.
```
It is possible that the problem come from before this line.

**Comment by yjacolin** [comment]
Your PR is really good, I just don't understand why test failed. Thanks a lot for your contribution!

Did you know if your PR is link to a ticket? Like this one: https://github.com/qgis/QGIS-Documentation/issues/583 If so you can add `Fix #583` to get a link between your commit and the ticket.

**Comment by purplexed** [comment]
@yjacolin Thank you for accepting me :)
There must be an error with the tabs and spaces. I will find it and push back again.
I will add the #852 in the commit as well.

**Comment by DelazJ** [comment]
@yjacolin wrote:
>Your PR is really good, I just don't understand why test failed. Thanks a lot for your contribution!

There's an indentation issue with the bulleted lists and their lines. I Proposed a quick PR on @purplexed  [branch](https://github.com/purplexed/QGIS-Documentation/pull/1)

**Comment by yjacolin** [comment]
@purplexed you should not add #582 (as it is your PR number) but the ticket number :) should be #583, isn't?

**Comment by purplexed** [comment]
@yjacolin oh yes, you are right...! Will do it in the next ones :)

**Comment by yjacolin** [comment]
Thanks!

**Pull request merged**